app/src/main/java/ingsw/group1/findmyphone/activity/NavHolderActivity.java

```diff
@@ -0,0 +1,32 @@
+package ingsw.group1.findmyphone.activity;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentActivity;
+
+import ingsw.group1.findmyphone.R;
+
+public class NavHolderActivity extends FragmentActivity {
+
+    private Fragment currentFragment = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'currentFragment' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4969671)